### DISCUSSION TITLE
[Kavita] Idea / Feature Submission - Send As enhancement

### DISCUSSION THREAD

**Discussion by Net0o** [Kareadita/Kavita#3406] [Ideas]
### Idea Description

1. Send As is a great feature for E-Readers like Kindle, however I am missing a feature which could send ebooks one-per-one from stacks/volumes.

E.g. I have above 100+ Series Volumes, it is time consuming to send one-per-one because the mail server can handle only 20MB+ files. However, when you try to send the whole Volume with 100+ e-books, it shows that the e-mail could not be sent because of the e-mail size limit.

Can be thing like this implemented in the future?

Thank you.

### Idea Category

Feature Enhancement

### Duration of Using Kavita

1.5 year

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by kariudo** [comment]
I'd like a send to webhook for this purpose, to make it easy to add an external service in the mix to handle things like merging volumes and converting to other formats in whatever workflow is needed to hit a target device. I.e. something like merge every 20 chapters as volumes, convert cbz to epub > push to reader etc. Without needing to expect functions like that in Kavita itself.
